### PR TITLE
Fix Examples For Block Usage

### PR DESCRIPTION
**Pull request by tgoodsell-tempus**
Based on `schema.Schema`, all `schema.Block` types must live in the `blocks` field instead of the examples showing their use in `attributes` at the top level.

### CODE REVIEW

File: website/docs/plugin/framework/handling-data/blocks/list-nested.mdx [hunk 1]
```diff
@@ -53,7 +53,8 @@ In this example, a resource schema defines a top level list nested block named `
 ```go
 func (r ThingResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
     resp.Schema = schema.Schema{
-        Attributes: map[string]schema.Attribute{
+        Attributes: map[string]schema.Attribute{}, // ... potentially other attributes ...
```
Comment (bendbennett) [inline website/docs/plugin/framework/handling-data/blocks/list-nested.mdx:56]:
```suggestion
```

File: website/docs/plugin/framework/handling-data/blocks/set-nested.mdx [hunk 2]
```diff
@@ -53,7 +53,8 @@ In this example, a resource schema defines a top level set nested block named `e
 ```go
 func (r ThingResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
     resp.Schema = schema.Schema{
-        Attributes: map[string]schema.Attribute{
+        Attributes: map[string]schema.Attribute{}, // ... potentially other attributes ...
```
Comment (bendbennett) [inline website/docs/plugin/framework/handling-data/blocks/set-nested.mdx:56]:
```suggestion
```

File: website/docs/plugin/framework/handling-data/blocks/single-nested.mdx [hunk 3]
```diff
@@ -48,7 +48,8 @@ In this example, a resource schema defines a top level single nested block named
 ```go
 func (r ThingResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
     resp.Schema = schema.Schema{
-        Attributes: map[string]schema.Attribute{
+        Attributes: map[string]schema.Attribute{}, // ... potentially other attributes ...
```
Comment (bendbennett) [inline website/docs/plugin/framework/handling-data/blocks/single-nested.mdx:51]:
```suggestion
```

File: website/docs/plugin/framework/handling-data/blocks/list-nested.mdx [hunk 4]
```diff
@@ -67,9 +68,8 @@ func (r ThingResource) Schema(ctx context.Context, req resource.SchemaRequest, r
                         },
                     },
                 },
-                // ... potentially other fields ...
```
Comment (bendbennett) [inline website/docs/plugin/framework/handling-data/blocks/list-nested.mdx:70]: This should probably remain, as it's possible to set other fields at this level (e.g., `CustomType`, `Description` etc)

**Review by bendbennett — COMMENTED** [review]
@tgoodsell-tempus thank you very much for the fixes.
I've made some minor suggestions.

File: website/docs/plugin/framework/handling-data/blocks/list-nested.mdx [hunk 5]
```diff
@@ -67,9 +67,8 @@ func (r ThingResource) Schema(ctx context.Context, req resource.SchemaRequest, r
                         },
                     },
                 },
```
Comment (tgoodsell-tempus) [inline website/docs/plugin/framework/handling-data/blocks/list-nested.mdx:69]:
```suggestion
                },
                // ... potentially other fields ...
```

**Comment by tgoodsell-tempus** [comment]
@bendbennett I've added your suggestions into the PR, LMK if there's any other modifications that would be good.

**Review by bendbennett — APPROVED** [review]
LGTM 🚀 

Many thanks for the contribution @tgoodsell-tempus

**Pull request merged**